src/Uno.Extensions.Core/Threading/FastAsyncLock.cs

```diff
@@ -0,0 +1,240 @@
+﻿// ******************************************************************
+// Copyright � 2015-2018 nventive inc. All rights reserved.
```

Comment:
Copyright?

Reply:
It has been copied from Uno.Core project https://github.com/unoplatform/Uno.Core/blob/master/src/Uno.Core.Extensions.Threading/FastAsyncLock.cs, as the file do have a copyright, I think that we should keep it.